vnext/Microsoft.ReactNative/Fabric/ActivityIndicatorComponentView.h

```diff
@@ -0,0 +1,46 @@
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "ComponentView.h"
+
+#include <Microsoft.ReactNative.Cxx/ReactContext.h>
+#include <UI.Xaml.Controls.h>
+#include "ViewComponentView.h"
+
+#pragma warning(push)
+#pragma warning(disable : 4244 4305)
```

Comment:
note 4244 is sdl, is this one that's already being fixed in core?

Reply:
Yes there is already a PR out for this one.